.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -317,9 +312,66 @@ stages:
         command: "Clean BuildProfilerHome"
 
     - publish: $(System.DefaultWorkingDirectory)/profiler/_build
```

Comment:
Not really, in fact the `profiler-home` is the published binaries and the `profiler-binaries` are all binaries (so files, tests binaries...)
The `binaries` are used for `unit_tests`.
If I compile profiler test binaries in the `unit_tests_linux` stage, it will recompile the whole profiler.

Reply:
> If I compile profiler test binaries in the unit_tests_linux stage, it will recompile the whole profiler.

Does it _have_ to though? 🤔 I mean, we don't for the Tracer, could we restructure it doesn't have to re-build everything? Doesn't have to be for this PR, but this stage may be on the critical path, so any reductions here may be useful.

Reply:
I saw the whole thing recompiling so :/
I did not dig into why it's rebuilding all the thing when we only want the test (timestamp issue ?)